packages/neocmakelsp/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Decodetalkers/neocmakelsp
+TERMUX_PKG_DESCRIPTION="the lsp for cmake"
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_MAINTAINER=@termux
+TERMUX_PKG_VERSION=0.7.1
```

Comment:
`0.7.2` and `0.7.3` have been released in the meantime.
Please update the initial version accordingly.
```suggestion
TERMUX_PKG_VERSION=0.7.3
```